beherabikram

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Sara Jaoude",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
+  "displayStatsPublic": true,
+  "bio": "I am a MERNStack developer",
+  "avatar": "https://avatars.githubusercontent.com/u/101280738?s=400&u=ab994845489a4322a33f2767f0ea7359f414b7b7&v=4",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/BikramB45875872",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/btw_its.bikram/",
+      "icon": "FaInstagram"
+    },
+       {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/beherabikram",
+      "icon": "FaGitHub"
+    },
+        {
+      "name": "Follow me on Linkdin",
+      "url": "https://www.linkedin.com/in/bikram-behera-4783b1259/",
+      "icon": "FaLInkdin"
+    },
```

Comment:
Consider removing the additional comma (**,**) it renders, the json file invalid
```suggestion
    }
```